package.json

```diff
@@ -14,6 +14,7 @@
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",
     "react": "^16.9.0",
+    "react-csv": "^1.1.2",
```

Comment:
Did this not result in any changes to `package-lock.json` as well?

Reply:
Added.